lib/ruby_warrior/decree.rb

```diff
@@ -0,0 +1,10 @@
+require 'active_record'
+
+module RubyWarrior
+  class Decree < ActiveRecord::Base
+    establish_connection adapter: "sqlite3", database: "warriors/foobar.db"
+    connection.create_table table_name, force: true do |t|
+      t.string :body
```

Comment:
What fields should go in here? Any validation?

Reply:
Level (number)
X (number)
Y (number)
Type (string. if possible the types would be out of a list, to make it easier validating correctness of the data)
Alive/Dead (Boolean, nullable)